src/Servers/Kestrel/Core/test/KestrelServerTests.cs

```diff
@@ -318,7 +318,7 @@ public async Task ListenIPWithEphemeralPort_TransportsGetIPv6Any()
         options.ApplicationServices = new ServiceCollection()
            .AddLogging()
            .BuildServiceProvider();
-        options.ListenAnyIP(0, options =>
+        options.ListenAnyIP(5000, options =>
```

Comment:
Given the name `ListenIPWithEphemeralPort_TransportsGetIPv6Any`, I think we should update this test to disable HTTP/3 instead.

Reply:
Duplicated this test but for HTTP/3 plus multiplex.